types/ratelimiter/ratelimiter-tests.ts

```diff
@@ -1,19 +1,16 @@
-import * as redis from 'redis';
-import * as ioredis from 'ioredis';
 import Limiter = require('ratelimiter');
 
 declare let id: string;
-declare let db: redis.RedisClient;
-declare let iodb: ioredis.Redis;
+declare let db: {
+    multi(operations: any[][]): { exec(cb: (err: any, res: any) => unknown): void };
+};
```

Comment:
Generally speaking, it's not so great to have dependencies just so they are used in tests, hence this removal.